torch/utils/data/datapipes/utils/common.py

```diff
@@ -20,6 +21,99 @@
 ]
 
 
+def validate_input_col(fn: Callable, input_col: Optional[Union[int, tuple, list]]):
+    """
+    Checks that function used in a map style datapipe works with the input column
+
+    This simply ensures that the number of keyword or positional arguments matches
```

Comment:
nit: `ensures that the number of positional arguments`.

We do not match any keyword argument.